core/integration/directory_test.go

```diff
@@ -1214,3 +1214,62 @@ func (DirectorySuite) TestDigest(ctx context.Context, t *testctx.T) {
 		require.Equal(t, "sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855", digest)
 	})
 }
+
+func (DirectorySuite) TestDirectoryName(ctx context.Context, t *testctx.T) {
+	c := connect(ctx, t)
+
+	t.Run("empty directory name", func(ctx context.Context, t *testctx.T) {
+		name, err := c.Directory().Name(ctx)
+		require.NoError(t, err)
+		require.Equal(t, "/", name)
+	})
+
+	t.Run("not found directory", func(ctx context.Context, t *testctx.T) {
+		_, err := c.Directory().Directory("foo").Name(ctx)
+		requireErrOut(t, err, "no such file or directory")
+	})
```

Comment:
Can you also a corresponding test in `file_test.go` for `File.name`? Would be good to ensure consistent behavior here.